r600_state: Fix SPI_THREAD_GROUPING setting

Seems to be inverted from what I guess is supposed. Register seems to be
r7xx specific. Anyway, xf86-video-ati and kernel drm set it to 1 for r7xx
and 0 for classes below.

diff --git a/src/gallium/drivers/r600/r600_state.c b/src/gallium/drivers/r600/r600_state.c
index 1be3e1b..6b31c58 100644
--- a/src/gallium/drivers/r600/r600_state.c
+++ b/src/gallium/drivers/r600/r600_state.c
@@ -2323,16 +2349,16 @@ void r600_init_atom_start_cs(struct r600_context *rctx)
 	if (rctx->b.chip_class >= R700) {
 		r600_store_context_reg(cb, R_028A50_VGT_ENHANCE, 4);
 		r600_store_config_reg(cb, R_008D8C_SQ_DYN_GPR_CNTL_PS_FLUSH_REQ, 0x00004000);
 		r600_store_config_reg(cb, R_009830_DB_DEBUG, 0);
 		r600_store_config_reg(cb, R_009838_DB_WATERMARKS, 0x00420204);
-		r600_store_context_reg(cb, R_0286C8_SPI_THREAD_GROUPING, 0);
+		r600_store_context_reg(cb, R_0286C8_SPI_THREAD_GROUPING, 1);
 	} else {
 		r600_store_config_reg(cb, R_008D8C_SQ_DYN_GPR_CNTL_PS_FLUSH_REQ, 0);
 		r600_store_config_reg(cb, R_009830_DB_DEBUG, 0x82000000);
 		r600_store_config_reg(cb, R_009838_DB_WATERMARKS, 0x01020204);
-		r600_store_context_reg(cb, R_0286C8_SPI_THREAD_GROUPING, 1);
+		r600_store_context_reg(cb, R_0286C8_SPI_THREAD_GROUPING, 0);
 	}
 	r600_store_context_reg_seq(cb, R_0288A8_SQ_ESGS_RING_ITEMSIZE, 9);
 	r600_store_value(cb, 0); /* R_0288A8_SQ_ESGS_RING_ITEMSIZE */
 	r600_store_value(cb, 0); /* R_0288AC_SQ_GSVS_RING_ITEMSIZE */
 	r600_store_value(cb, 0); /* R_0288B0_SQ_ESTMP_RING_ITEMSIZE */
